index.md

```diff
@@ -73,9 +73,10 @@ Bitte beachtet die folgenden Schritte:
     - Bei Problemen können wir gerne per [Mail](about.html) oder persönlich helfen
 - Einen Laptop mitbringen, eine begrenzte Anzahl Laptops kann auch von uns gestellt werden
 - Verlängerungskabel/Merfachstecker mitbringen (falls möglich)
```

Comment:
Could you fix the *Merfachstecker* while you are at it? 😅

Reply:
Fixed it using the webpage ;)